# Homework  - The Best Soccer Player

### In this exercise you will find the best socces player, according to fifa ranking
You will do this using your python knowledge about variables and lists!
Remember: Press the Run button to execute code

In [19]:
# just execute this code, you don't have to understand it (yet;)
import pandas as pd
df = pd.read_csv("fifa.csv")
name = list(df.iloc[:,1])
rating = list(df.iloc[:,2])
height = list(df.iloc[:,4])
foot = list(df.iloc[:,5])

We have 4 python lists with data about soccer players, with the following names: "name", "rating", "height", "foot". 

In "name", we got all the names of the players. Rating defines how good a player is. We also have the player's height and his preference for playing with his Right or Left foot.

Lets look at the "name" list:

In [27]:
name[0:5]

[' Gábor Király',
 ' Frederik Boi',
 ' Tomasz Szewczuk',
 ' Steeve Joseph-Reinette',
 ' Kamel Chafni']

In [28]:
# show the first 5 entries of the rating list

With the max() function, we find the maximum of a list. For example, if we want the height (in cm) of the tallest person:

In [29]:
max(height)

208

In [23]:
# to-do: find the highest rating in the 'rating' list and save this to the variable 'max_rating'
max_rating = 

We can do a lot of stuff with lists in Python. With the following line of code, we find the place in the list (index) where the rating is the highest. You don't have to fully understand this line of code yet. Just run it for now! :)

In [30]:
index_best_player = rating.index(max_rating)
index_best_player

2195

So player number 2195 is the best player. We just need to know what his name is!

In [31]:
# To-do: What is worlds best soccer player?

In [32]:
# To-do: Does he plays with his Right or Left foot?